### DISCUSSION TITLE
"Taskbar height and icon size" installation should restart Windows Explorer

### DISCUSSION THREAD

**Discussion by dhb33** [ramensoftware/windhawk-mods#824] [Q&A] [answered]
Because my taskbar is on a 200% (Recommended) DPI, its height got cut by half after install. Restarting Windows Explorer solved the issue. Now, Disable/Enable doesn't break it anymore.  Great!

**Comment by m417z** [comment]
Sounds like this issue: https://github.com/ramensoftware/windhawk-mods/issues/529

I don't think it's related to DPI, I tried reproducing it in various ways, but didn't find a way to reproduce it, so it's difficult to diagnose and fix. Just tried again with high DPI but it worked correctly for me. If you have a way to reproduce the issue, please let me know.

**Reply by dhb33** [reply, marked as answer]
I couldn't preproduce it. It happened before I installed the KB5040527. Soo, I'll close this discussion as outdated.
